cloud/ec2_client.go

```diff
@@ -985,6 +1013,30 @@ func (c *awsClientImpl) AssociateAddress(ctx context.Context, input *ec2.Associa
 	return output, nil
 }
 
+func (c *awsClientImpl) DisassociateAddress(ctx context.Context, input *ec2.DisassociateAddressInput) (*ec2.DisassociateAddressOutput, error) {
+	var output *ec2.DisassociateAddressOutput
+	var err error
+	err = utility.Retry(
+		ctx,
+		func() (bool, error) {
+			msg := makeAWSLogMessage("DisassociateAddress", fmt.Sprintf("%T", c), input)
+			output, err = c.ec2Client.DisassociateAddress(ctx, input)
+			if err != nil {
+				var apiErr smithy.APIError
+				if errors.As(err, &apiErr) {
+					grip.Debug(message.WrapError(apiErr, msg))
+				}
+				return true, err
+			}
+			grip.Info(msg)
+			return false, nil
+		}, awsClientDefaultRetryOptions())
+	if err != nil {
+		return nil, err
+	}
+	return output, nil
```

Comment:
small nit (and I noticed other functions have this same thing) but can we just `return output, err`?

Reply:
I'm a bit inclined to just leave it because they are the same and this makes it a little more explicit that you get back either a valid output xor the error but not both from the AWS API.

Reply:
Sure, sounds good